<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>273</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>273</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>273</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rc.qrc">
    <normaloff>:/ico/ed</normaloff>:/ico/ed</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageLogin">
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>451</width>
          <height>81</height>
         </rect>
        </property>
        <property name="title">
         <string>服务器配置信息：</string>
        </property>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>264</x>
           <y>42</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>服务器端口：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>42</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>服务器地址：</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="serPort">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>36</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="minimum">
          <number>1000</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
         <property name="value">
          <number>2048</number>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineServer">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>37</y>
           <width>131</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>127.0.0.1</string>
         </property>
        </widget>
       </widget>
       <widget class="QSpinBox" name="loaPort">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>223</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1000</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>6666</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>254</x>
          <y>230</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>本地监听端口：</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>90</y>
          <width>451</width>
          <height>131</height>
         </rect>
        </property>
        <property name="title">
         <string>登录信息：</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>62</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;用户名：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineUser">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>40</y>
           <width>231</width>
           <height>31</height>
          </rect>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>80</y>
           <width>62</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;密    码：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="linepword">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>80</y>
           <width>231</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
        <widget class="QPushButton" name="butLogin">
         <property name="geometry">
          <rect>
           <x>340</x>
           <y>40</y>
           <width>101</width>
           <height>71</height>
          </rect>
         </property>
         <property name="text">
          <string>登录</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="pageState">
       <widget class="QPushButton" name="butHidde">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>180</y>
          <width>191</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>隐藏窗口</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
